### PR TITLE
SDKS-2992 Build pipeline for ping-protect module

### PR DESCRIPTION
**Pull request by spetrov**
# JIRA Ticket

[SDKS-2992](https://bugster.forgerock.org/jira/browse/SDKS-2992) Build pipeline for ping-protect module

# Description
- Introduced a new module for all e2e tests `forgerock-integration-tests` (SDKS-2964)
- Updated the GitHub actions accordingly

### CODE REVIEW

File: .github/workflows/bitbar-prepare-artifacts.yaml [hunk 1]
```diff
@@ -46,23 +46,23 @@ jobs:
       - name: List build tools versions
         run: ls /Users/runner/Library/Android/sdk/build-tools/
 
-      # Sign auth-debug-androidTest.apk
-      - name: Sign auth-debug-androidTest.apk
+      # Sign app-debug-androidTest.apk
```
Comment (spetrov) [inline .github/workflows/bitbar-prepare-artifacts.yaml:49]: We plan to retire the `auth` sample app... BitBar however requires an app in order to execute the e2e tests, so I switched to the `app` sample instead...

File: .github/workflows/bitbar-prepare-artifacts.yaml [hunk 2]
```diff
@@ -46,23 +46,23 @@ jobs:
...
           alias: ${{ secrets.SIGNING_ALIAS }}
           keyStorePassword: ${{ secrets.SIGNING_KEYSTORE_PASSWORD }}
           keyPassword: ${{ secrets.SIGNING_KEY_PASSWORD }}
         env:
           BUILD_TOOLS_VERSION: "34.0.0"
 
-      # Sign forgerock-auth-debug-androidTest.apk
-      - name: Sign forgerock-auth-debug-androidTest.apk
+      # Sign forgerock-integration-tests-debug-androidTest.apk
+      - name: Sign forgerock-integration-tests-debug-androidTest.apk
```
Comment (spetrov) [inline .github/workflows/bitbar-prepare-artifacts.yaml:62]: The e2e tests are now in the `forgerock-integration-tests` module... This module includes all e2e tests (`forgerock-core`, `forgerock-auth`, `ping-protect`, `forgerock-authenticator`)

File: .github/workflows/build-and-test.yaml [hunk 3]
```diff
@@ -43,13 +43,17 @@ jobs:
       - name: Run forgerock-authenticator debug unit tests
         run: ./gradlew :forgerock-authenticator:testDebugUnitTest --stacktrace --no-daemon
 
+      # Execute forgerock-authenticator debug unit tests
+      - name: Run ping-protect debug unit tests
+        run: ./gradlew :ping-protect:testDebugUnitTest --stacktrace --no-daemon
```
Comment (spetrov) [inline .github/workflows/build-and-test.yaml:48]: Added the `ping-protect` unit tests along with the other modules tests...

File: .github/workflows/run-live-tests.yaml [hunk 4]
```diff
@@ -6,7 +6,7 @@ on:
         description: The AM url to run live test cases against
         type: string
         required: true
-        default: https://openam-forgerrock-sdksteanant.forgeblocks.com/am
```
Comment (spetrov) [inline .github/workflows/run-live-tests.yaml:9]: `openam-forgerrock-sdksteanant` tenant is no longer available...

File: .vscode/settings.json
Comment (spetrov) [inline .vscode/settings.json:1]: I'll remove this file... I added it by mistake...

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/AppPinAuthenticatorTest.kt [hunk 5]
```diff
@@ -35,10 +37,19 @@ class AppPinAuthenticatorTest {
...
-    @Test(expected = UnrecoverableKeyException::class)
+    @Test
     fun testInvalidPin() {
         appPinAuthenticator.generateKeys(context, "1234".toCharArray())
-        appPinAuthenticator.getPrivateKey(context, "invalid".toCharArray())
+        try {
+            appPinAuthenticator.getPrivateKey(context, "invalid".toCharArray())
+            fail()
+        }
+        catch (e: UnrecoverableKeyException) {
```
Comment (spetrov) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/AppPinAuthenticatorTest.kt:47]: Updated this test together with Jey

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/FRDeviceProfileTest.java [hunk 6]
```diff
@@ -99,13 +104,34 @@ public void testDeviceProfile() throws JSONException, ExecutionException, Interr
...
         // or if ACCESS_BACKGROUND_LOCATION permission is not granted
-        int backgroundLocationPermissionApproved =
-                ActivityCompat.checkSelfPermission(context,
-                        Manifest.permission.ACCESS_BACKGROUND_LOCATION);
+        boolean locationPermissionGranted = false;
 
-        if (!isEmulator() && backgroundLocationPermissionApproved >= 0) {
-            result.getJSONObject("location").getDouble("latitude");
-            result.getJSONObject("location").getDouble("longitude");
+        // If the location services are NOT enabled skip the rest of the test...
```
Comment (spetrov) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/FRDeviceProfileTest.java:109]: Attempted to fix random failures on BitBar where for some reason on some divide location data is not available...

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/AppIntegrityCallbackTest.java [hunk 7]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022 - 2024 ForgeRock. All rights reserved.
+ * Copyright (c) 2022 - 2023 ForgeRock. All rights reserved.
```
Comment (spetrov) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/AppIntegrityCallbackTest.java:2]: 2023 is the correct year here... this test was moved from `auth` module to `integration-tests`

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/BaseDeviceBindingTest.java [hunk 8]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2023 - 2024 ForgeRock. All rights reserved.
+ * Copyright (c) 2023 ForgeRock. All rights reserved.
```
Comment (spetrov) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/BaseDeviceBindingTest.java:2]: 2023 is the correct year here... this test was moved from `auth` module to `integration-tests`

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/DeviceProfileCollectorCallbackAndroidTest.java [hunk 9]
```diff
@@ -111,17 +122,37 @@ public void testLocation() throws Exception {
...
         assertTrue(content.contains("identifier"));
 
-        int backgroundLocationPermissionApproved =
-                ActivityCompat.checkSelfPermission(context,
-                        Manifest.permission.ACCESS_BACKGROUND_LOCATION);
+        boolean locationPermissionGranted = false;
+
 
-        if (!isEmulator() && backgroundLocationPermissionApproved >= 0) {
+        // If the location services are NOT enabled skip the rest of the test...
```
Comment (spetrov) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/DeviceProfileCollectorCallbackAndroidTest.java:128]: Attempted to fix random failures on BitBar where for some reason on some divide location data is not available...

File: ping-protect/gradle.properties [hunk 10]
```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2021 ForgeRock. All rights reserved.
```
Comment (spetrov) [inline ping-protect/gradle.properties:2]: Fix copyright header

File: samples/app/src/main/java/com/example/app/callback/PingOneProtectInitCallback.kt [hunk 11]
```diff
@@ -24,12 +24,12 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.unit.dp
 import org.forgerock.android.auth.Logger
-import org.forgerock.android.auth.PingOneProtectInitCallback
+import org.forgerock.android.auth.PingOneProtectInitializeCallback
 import org.forgerock.android.auth.PingOneProtectInitException
 
 @Composable
 fun PingOneProtectInitCallback(
```
Comment (witrisna) [inline samples/app/src/main/java/com/example/app/callback/PingOneProtectInitCallback.kt:31]: To align with other, please rename it to PingOneProtectInitializeCallback

**Review by witrisna — CHANGES_REQUESTED** [review]

File: samples/app/src/main/java/com/example/app/callback/PingOneProtectInitCallback.kt (hunk 11 above)
Comment (spetrov) [reply samples/app/src/main/java/com/example/app/callback/PingOneProtectInitCallback.kt:31]: 👍🏻  Renamed...

File: gradle/libs.versions.toml [hunk 12]
```diff
@@ -74,7 +77,9 @@ org-robolectric-robolectric = { group = "org.robolectric", name = "robolectric",
 espresso-core = { group = "androidx.test.espresso", name = "espresso-core", version.ref = "espresso-core" }
 io-mockk = { group = "io.mockk", name = "mockk", version.ref = "io-mockk" }
 org-jetbrains-kotlinx = { group = "org.jetbrains.kotlinx", name = "kotlinx-coroutines-android", version.ref = "org-jetbrains-kotlinx" }
-com-pingidentity-signals = { group = "com.pingidentity.signals", name = "android-sdk", version.ref = "com-pingidentity-signals" }
+com-pingidentity-signals  = { group = "com.pingidentity.signals", name = "android-sdk", version.ref = "com-pingidentity-signals" }
+appcompat = { group = "androidx.appcompat", name = "appcompat", version.ref = "appcompat" }
```
Comment (jeyanthanperiyasamy) [inline gradle/libs.versions.toml:81]: can you check this both needed or not...

File: ping-protect/build.gradle.kts [hunk 13]
```diff
@@ -8,40 +8,50 @@
...
+apply<AndroidBuildGradlePlugin>()
+
 android {
     namespace = "org.forgerock.android.protect"
-    compileSdk = 34
+}
 
-    defaultConfig {
-        minSdk = 23
+tasks.dokkaHtml.configure {
```
Comment (jeyanthanperiyasamy) [inline ping-protect/build.gradle.kts:23]: at some point we need to move this to its own dokka plugin class

File: ping-protect/build.gradle.kts [hunk 14]
```diff
@@ -8,40 +8,50 @@
 plugins {
     alias(libs.plugins.androidLibrary)
     alias(libs.plugins.kotlinAndroid)
+    id("com.adarshr.test-logger")
+    id("maven-publish")
+    id("signing")
+    id("org.jetbrains.dokka")
 }
 
+apply<AndroidBuildGradlePlugin>()
```
Comment (jeyanthanperiyasamy) [inline ping-protect/build.gradle.kts:17]: good..now all the projects aligned to this AndroidBuildGradlePlugin

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/AppPinAuthenticatorTest.kt [hunk 15]
```diff
@@ -4,6 +4,8 @@ import android.content.Context
 import androidx.test.core.app.ApplicationProvider
 import androidx.test.ext.junit.runners.AndroidJUnit4
 import org.assertj.core.api.Assertions
+import org.junit.Assert.fail
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/AppPinAuthenticatorTest.kt:7]: Missing copyright

File: forgerock-integration-tests/build.gradle.kts [hunk 16]
```diff
@@ -0,0 +1,46 @@
+
```
Comment (rodrigoareis) [inline forgerock-integration-tests/build.gradle.kts:1]: Missing copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/DefaultTokenManagerTest.java [hunk 17]
```diff
@@ -7,9 +7,14 @@
 
 package org.forgerock.android.auth;
 
+import static org.junit.Assert.assertEquals;
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/DefaultTokenManagerTest.java:10]: Update copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/FRDeviceProfileTest.java [hunk 18]
```diff
@@ -13,14 +13,16 @@
 
 import android.Manifest;
 import android.content.Context;
+import android.content.pm.PackageManager;
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/FRDeviceProfileTest.java:16]: Update copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/FRSessionTest.java [hunk 19]
```diff
@@ -7,6 +7,9 @@
 
 package org.forgerock.android.auth;
 
+import static org.assertj.core.api.Assertions.assertThat;
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/FRSessionTest.java:10]: Revert change or update copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/PlatformUsernamePasswordNodeListener.java [hunk 20]
```diff
@@ -7,14 +7,14 @@
 
 package org.forgerock.android.auth;
 
+import static org.forgerock.android.auth.AndroidBaseTest.PASSWORD;
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/PlatformUsernamePasswordNodeListener.java:10]: Revert change or update copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/SetPersistentCookieTest.java [hunk 21]
```diff
@@ -7,13 +7,13 @@
 
 package org.forgerock.android.auth;
 
+import static org.assertj.core.api.Assertions.assertThat;
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/SetPersistentCookieTest.java:10]: Revert change or update copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/TreeTest.java [hunk 22]
```diff
@@ -9,8 +9,6 @@
 
 import androidx.test.ext.junit.runners.AndroidJUnit4;
 
-import org.forgerock.android.auth.callback.NameCallback;
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/TreeTest.java:12]: Update copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/UsernamePasswordNodeListener.java [hunk 23]
```diff
@@ -7,17 +7,14 @@
 
 package org.forgerock.android.auth;
 
+import static org.forgerock.android.auth.AndroidBaseTest.PASSWORD;
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/UsernamePasswordNodeListener.java:10]: Update copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/BooleanAttributeInputCallbackTest.java [hunk 24]
```diff
@@ -7,19 +7,17 @@
 
 package org.forgerock.android.auth.callback;
 
-import org.assertj.core.api.Assertions;
+import static org.assertj.core.api.Assertions.assertThat;
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/BooleanAttributeInputCallbackTest.java:10]: update copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/ChoiceCallbackTest.java [hunk 25]
```diff
@@ -7,16 +7,16 @@
 
 package org.forgerock.android.auth.callback;
 
-import org.forgerock.android.auth.TreeTest;
+import static org.assertj.core.api.Assertions.assertThat;
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/ChoiceCallbackTest.java:10]: Revert change or update copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/ConfirmationCallbackTest.java [hunk 26]
```diff
@@ -7,16 +7,16 @@
 
 package org.forgerock.android.auth.callback;
 
-import org.forgerock.android.auth.TreeTest;
+import static org.assertj.core.api.Assertions.assertThat;
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/ConfirmationCallbackTest.java:10]: Revert change or update copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/ConsentMappingCallbackTest.java [hunk 27]
```diff
@@ -7,6 +7,8 @@
 
 package org.forgerock.android.auth.callback;
 
+import static org.assertj.core.api.Assertions.assertThat;
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/ConsentMappingCallbackTest.java:10]: Revert change or update copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/DeviceProfileCollectorCallbackAndroidTest.java [hunk 28]
```diff
@@ -7,13 +7,19 @@
 
 package org.forgerock.android.auth.callback;
 
+import static android.Manifest.permission.ACCESS_COARSE_LOCATION;
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/DeviceProfileCollectorCallbackAndroidTest.java:10]: Ppdate copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/KbaCreateCallbackTest.java [hunk 29]
```diff
@@ -7,18 +7,17 @@
 
 package org.forgerock.android.auth.callback;
 
+import static org.assertj.core.api.Assertions.assertThat;
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/KbaCreateCallbackTest.java:10]: Update copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/KeyAttestationTest.java [hunk 30]
```diff
@@ -33,7 +33,6 @@
 import org.junit.runner.RunWith;
 
 import java.text.ParseException;
-import java.util.Collections;
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/KeyAttestationTest.java:36]: Update copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/NameCallbackTest.java [hunk 31]
```diff
@@ -7,16 +7,16 @@
 
 package org.forgerock.android.auth.callback;
 
-import org.forgerock.android.auth.TreeTest;
+import static org.assertj.core.api.Assertions.assertThat;
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/NameCallbackTest.java:10]: Revert change or update copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/NumberAttributeInputCallbackTest.java [hunk 32]
```diff
@@ -7,19 +7,16 @@
 
 package org.forgerock.android.auth.callback;
 
-import org.assertj.core.api.Assertions;
+import static org.assertj.core.api.Assertions.assertThat;
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/NumberAttributeInputCallbackTest.java:10]: Update copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/PageCallback65Test.java [hunk 33]
```diff
@@ -7,6 +7,8 @@
 
 package org.forgerock.android.auth.callback;
 
+import static org.assertj.core.api.Assertions.assertThat;
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/PageCallback65Test.java:10]: Revert change or update copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/PageCallbackTest.java [hunk 34]
```diff
@@ -7,6 +7,8 @@
 
 package org.forgerock.android.auth.callback;
 
+import static org.assertj.core.api.Assertions.assertThat;
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/PageCallbackTest.java:10]: Revert change or update copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/PasswordCallbackTest.java [hunk 35]
```diff
@@ -7,16 +7,16 @@
 
 package org.forgerock.android.auth.callback;
 
-import org.forgerock.android.auth.TreeTest;
+import static org.assertj.core.api.Assertions.assertThat;
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/PasswordCallbackTest.java:10]: Revert change or update copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/PollingWaitCallbackTest.java [hunk 36]
```diff
@@ -7,16 +7,16 @@
 
 package org.forgerock.android.auth.callback;
 
-import org.forgerock.android.auth.TreeTest;
+import static org.assertj.core.api.Assertions.assertThat;
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/PollingWaitCallbackTest.java:10]: Revert change or update copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/ReCaptchaCallbackTest.java [hunk 37]
```diff
@@ -7,18 +7,18 @@
 
 package org.forgerock.android.auth.callback;
 
-import org.forgerock.android.auth.TreeTest;
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/ReCaptchaCallbackTest.java:10]: Revert change or update copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/StringAttributeInputCallbackTest.java [hunk 38]
```diff
@@ -7,6 +7,8 @@
 
 package org.forgerock.android.auth.callback;
 
+import static org.assertj.core.api.Assertions.assertThat;
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/StringAttributeInputCallbackTest.java:10]: Revert change or update copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/TermsAndConditionCallbackTest.java [hunk 39]
```diff
@@ -7,6 +7,8 @@
 
 package org.forgerock.android.auth.callback;
 
+import static org.assertj.core.api.Assertions.assertThat;
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/TermsAndConditionCallbackTest.java:10]: Revert change or update copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/ValidatedCreateUsernameCallbackTest.java [hunk 40]
```diff
@@ -7,15 +7,15 @@
 
 package org.forgerock.android.auth.callback;
 
+import static org.assertj.core.api.Assertions.assertThat;
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/ValidatedCreateUsernameCallbackTest.java:10]: Revert change or update copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/collector/FRDeviceIdentifierTest.java [hunk 41]
```diff
@@ -7,8 +7,14 @@
 
 package org.forgerock.android.auth.collector;
 
+import static org.forgerock.android.auth.Encryptor.ANDROID_KEYSTORE;
```
Comment (rodrigoareis) [inline forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/collector/FRDeviceIdentifierTest.java:10]: Update copyright

File: forgerock-integration-tests/build.gradle.kts (hunk 16 above)
Comment (spetrov) [reply forgerock-integration-tests/build.gradle.kts:1]: Added copyright.

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/AppPinAuthenticatorTest.kt (hunk 15 above)
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/AppPinAuthenticatorTest.kt:7]: Added copyright note

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/DefaultTokenManagerTest.java (hunk 17 above)
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/DefaultTokenManagerTest.java:10]: Reverted the changes and left the original copyright note...

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/FRDeviceProfileTest.java (hunk 18 above)
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/FRDeviceProfileTest.java:16]: Updated

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/FRSessionTest.java (hunk 19 above)
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/FRSessionTest.java:10]: Reverted the changes and left the original copyright note...

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/FRUserTest.java [hunk 42]
```diff
@@ -7,6 +7,10 @@
 
 package org.forgerock.android.auth;
 
+import static org.junit.Assert.assertEquals;
```
[the remark this answers is not in the document]
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/FRUserTest.java:10]: Reverted the changes and left the original copyright note...

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/PlatformUsernamePasswordNodeListener.java (hunk 20 above)
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/PlatformUsernamePasswordNodeListener.java:10]: Reverted the changes and left the original copyright note...

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/SecuredStorageTest.java [hunk 43]
```diff
@@ -7,6 +7,11 @@
 
 package org.forgerock.android.auth;
 
+import static org.junit.Assert.assertEquals;
```
[the remark this answers is not in the document]
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/SecuredStorageTest.java:10]: Reverted the changes

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/SetPersistentCookieTest.java (hunk 21 above)
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/SetPersistentCookieTest.java:10]: Reverted

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/TreeTest.java (hunk 22 above)
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/TreeTest.java:12]: Updated

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/UsernamePasswordNodeListener.java (hunk 23 above)
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/UsernamePasswordNodeListener.java:10]: Updated

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/BooleanAttributeInputCallbackTest.java (hunk 24 above)
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/BooleanAttributeInputCallbackTest.java:10]: Updated

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/ConfirmationCallbackTest.java (hunk 26 above)
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/ConfirmationCallbackTest.java:10]: Reverted

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/ConsentMappingCallbackTest.java (hunk 27 above)
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/ConsentMappingCallbackTest.java:10]: Reverted

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/DeviceProfileCallbackTest.java [hunk 44]
```diff
@@ -7,19 +7,19 @@
 
 package org.forgerock.android.auth.callback;
 
+import static org.assertj.core.api.Assertions.assertThat;
```
[the remark this answers is not in the document]
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/DeviceProfileCallbackTest.java:10]: Reverted

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/DeviceProfileCollectorCallbackAndroidTest.java (hunk 28 above)
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/DeviceProfileCollectorCallbackAndroidTest.java:10]: Updated copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/KeyAttestationTest.java (hunk 30 above)
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/KeyAttestationTest.java:36]: Updated copyright

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/NameCallbackTest.java (hunk 31 above)
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/NameCallbackTest.java:10]: Reverted

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/NumberAttributeInputCallbackTest.java (hunk 32 above)
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/NumberAttributeInputCallbackTest.java:10]: Updated

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/PageCallback65Test.java (hunk 33 above)
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/PageCallback65Test.java:10]: Reverted

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/PasswordCallbackTest.java (hunk 35 above)
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/PasswordCallbackTest.java:10]: Reverted

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/PollingWaitCallbackTest.java (hunk 36 above)
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/PollingWaitCallbackTest.java:10]: Reverted

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/ReCaptchaCallbackTest.java (hunk 37 above)
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/ReCaptchaCallbackTest.java:10]: Reverted

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/StringAttributeInputCallbackTest.java (hunk 38 above)
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/StringAttributeInputCallbackTest.java:10]: Reverted

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/TermsAndConditionCallbackTest.java (hunk 39 above)
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/TermsAndConditionCallbackTest.java:10]: Reverted

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/ValidatedCreateUsernameCallbackTest.java (hunk 40 above)
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/ValidatedCreateUsernameCallbackTest.java:10]: Reverted

File: forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/collector/FRDeviceIdentifierTest.java (hunk 41 above)
Comment (spetrov) [reply forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/collector/FRDeviceIdentifierTest.java:10]: Updated copyright

File: gradle/libs.versions.toml (hunk 12 above)
Comment (spetrov) [reply gradle/libs.versions.toml:81]: Yes, looks like both `appcompat` and `material` are needed

**Review by rodrigoareis — APPROVED** [review]
Changes looks good to me

**Pull request merged**